packages/mobile/fastlane/Fastfile

```diff
@@ -89,9 +89,9 @@ platform :ios do
   desc 'Prepare TestFlight'
   lane :prepare_testflight_and_upload do |options|
     api_key = app_store_connect_api_key(
-      key_id: '3NK3HR2Q69',
+      key_id: '3B5PGG27Q4',
```

Comment:
is this paired with a secret?  if so and if possible (and probably for another new PR) it makes sense to store this along side it's secret in secret manager.  i.e., so the "public" (or whatever this value is) and the "private" (or whatever the secret is) are in the same secret and can be rotated atomically together.

Reply:
Sure! Good idea. Will do that in a separate PR.